src/lightgraph_extensions.jl

```diff
@@ -59,7 +59,8 @@ end
 
 """ Removes multiple vertices with given Indices at once"""
 function rem_vertices!(EG::AbstractEmbeddedGraph, vs::AbstractVector{<:Integer}, args...)
-    for i in vs
+    vs_sorted = sort(vs, rev=true)    
```

Comment:
we should use the mutating `sort!` here, instead of allocating a new array `vs_sorted`